addons/web/static/src/views/pivot/pivot_view.js

```diff
@@ -19,6 +19,7 @@ export const pivotView = {
     ArchParser: PivotArchParser,
     SearchModel: PivotSearchModel,
     searchMenuTypes: ["filter", "groupBy", "comparison", "favorite"],
+    buttonTemplate: "web.PivotView.Buttons",
```

Comment:
Coucou @aab-odoo 

FYI, I added `buttonTemplate` in pivotView as we discussed on discord last time. 🙂